accounts-db/src/bucket_map_holder.rs

```diff
@@ -61,6 +61,9 @@ pub struct BucketMapHolder<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>>
     /// how many ages should elapse from the last time an item is used where the item will remain in the cache
     pub ages_to_stay_in_cache: Age,
 
+    /// estimated total number of accounts for calculating per-bin HashMap capacity
```

Comment:
nit: Can these comments replace "estimated" with "initial" too?

Something like (which looks like the help text):

```suggestion
    /// initial number of accounts, used to pre-allocate the in-mem index
```

(but feel free to push back and/or reword)